desktop/libs/metadata/src/metadata/assistant/data/queries.json

```diff
@@ -1,81 +1,85 @@
+        "statement": "use clusterstats;\nSELECT count(DISTINCT(clusterid))\nFROM cluster_config\nWHERE servicetype = 'HDFS'\n  AND configname = 'hdfs_sentry_sync_enable'\n  AND configvalue = 'true';"
       }
     }
   },
   {
-    "name": "Sample: Top salary",
-    "desc": "Top salary 2007 above $100k",
-    "dialects": [],
+    "name": "Escalation Jira analysis",
+    "desc": "Investigate quality of escalation responses",
+    "dialects": [
+      "impala"
+    ],
     "data": {
       "query": {
-        "statement": "SELECT sample_07.description, sample_07.salary\r\nFROM\r\n  sample_07\r\nWHERE\r\n( sample_07.salary > 100000)\r\nORDER BY sample_07.salary DESC\r\nLIMIT 1000"
+        "statement": "SELECT trunc( from_unixtime(cast( createdts / 1000 AS int)), '${bucket}') AS q, -- Q, MONTH\n   count(*) as ct,\n   cast(avg(duration) AS int) avg_day,\n   min(case when cumedist >= 0.33 then duration end) as 33percentile,\n   min(case when cumedist >= 0.50 then duration end) as 50percentile,\n   min(case when cumedist >= 0.80 then duration end) as 80percentile,\n   min(case when cumedist >= 0.90 then duration end) as 90percentile\nFROM\n( select \n    createdts,\n    cast( (resolvedts - createdts) / 1000 / 60 / 60 / 24 AS int) as duration,\n    cume_dist() over (partition by trunc( from_unixtime(cast( createdts / 1000 AS int)), '${bucket}') order by cast( (resolvedts - createdts) / 1000 / 60 / 60 / 24 AS int) asc) as cumedist \n       from jira.ticket \n       WHERE LOWER(componentsasstr) LIKE LOWER('%${component}%')\n        AND resolvedts != 0\n        AND type.value = '${type}'\n        AND createdts >= unix_timestamp('${date}', 'yyyy-MM-dd') * 1000\n        --AND from_unixtime(cast( createdts / 1000 AS int)) >= toun('${date}', 'UTC')\n       -- AND NOT LOWER(title) like '%backport%' \n       -- AND NOT LOWER(title) like '%patch%'\n      ) as t\n\nGROUP BY  q\nORDER BY  q asc\n;"
       }
     }
   },
   {
-    "name": "Sample: Salary growth",
-    "desc": "Salary growth (sorted) from 2007-08",
-    "dialects": [],
+    "name": "Escalation Jira analysis: List of escalations",
+    "desc": "Investigate quality of escalation responses",
+    "dialects": [
+      "impala"
+    ],
     "data": {
       "query": {
-        "statement": "SELECT s07.description, s07.salary, s08.salary,\r\n  s08.salary - s07.salary\r\nFROM\r\n  sample_07 s07 JOIN sample_08 s08\r\nON ( s07.code = s08.code)\r\nWHERE\r\n s07.salary < s08.salary\r\nORDER BY s08.salary-s07.salary DESC\r\nLIMIT 1000"
+        "statement": "SELECT from_unixtime(cast( createdts / 1000 AS int)) as t,\nissuekey,\n       summary, resolutionname, statusname\nFROM jira.ticket\nWHERE LOWER(componentsasstr) LIKE LOWER('%${component}%') --AND resolvedts != 0\n  AND type.value = '${type}'\n  AND createdts >= unix_timestamp('${date}', 'yyyy-MM-dd') * 1000\n  order by createdts desc, issuekey desc\n  ;"
```

Comment:
Note: we can provide default values to the params, e.g. `${component=hue}`
if not the user will be stuck

Reply:
Added default values to the parameters according to:

<img width="797" alt="Screenshot 2021-06-07 at 3 35 36 PM" src="https://user-images.githubusercontent.com/42064744/120998769-0528f180-c7a6-11eb-9622-337d2d62c2c5.png">